scripts/setup.sh

```diff
@@ -1,30 +1,52 @@
 #!/bin/bash
 
+source ${topdir}/scripts/common.sh
+
+function setup_log_file() {
+	# create the log directory if it doesn't already exist
+	LOG_DIR="${topdir}/logs"
+	mkdir -p "${LOG_DIR}"
+
+	# we want to create a different log file for each run. So name these files
+	# with timestamps.
+	export LOG_FILE="$LOG_DIR/run_$(date +"%Y-%m-%d_%H:%M:%S").log"
```

Comment:
better to have build name with the log file instead or along with the timestamps.

Reply:
In "sudo ./build.sh am62xx-evm", is the build name "am62xx-evm"? If yes, what should the name be if the user doesn't specify any arguments ("all"?)?

Reply:
the readme is outdated. My bad. 
It's not `sudo ./build.sh <machine>` anymore. It is `sudo ./build.sh <build>`.
The <build> is what you have in "builds.toml" file.
So I suggest log file to be `logs/${build}.log` or `logs/${build}_<timestamp>.log`

FYI, we'll not build all if no arg is specified. That will change in the README and build.sh update that I'm gonna do today. Coz no one will want to build all supported build configs.

The Github Workflow follows the same approach.